packages/terra-tag/tests/jest/Tag.test.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Tag from '../../src/Tag';
+
+describe('Tag', () => {
+  const defaultRender = <Tag text="default" />;
+
+  // Snapshot Tests
+  it('should render a tag', () => {
+    const tag = shallow(<Tag text="Normal" />);
+    expect(tag).toMatchSnapshot();
+  });
+
+  it('should render as an anchor when provided an href', () => {
+    const tag = shallow(<Tag text="href" href="MockHref" />);
+    expect(tag).toMatchSnapshot();
+  });
+
+  it('should render an icon and a text', () => {
+    const testElement = <img alt="icon" />;
+    const tag = shallow(<Tag icon={testElement} text="text" />);
+    expect(tag).toMatchSnapshot();
+  });
+
+  // Prop Tests
+  it('should have the class is-interactive', () => {
+    const tag = shallow(<Tag text="text" onClick={() => 1} />);
+    expect(tag.prop('className')).toContain('tag is-interactive');
```

Comment:
Do a screenshot test here instead. That will give you better test coverage.

Reply:
And for the rest of theses tests, if feels like matching against a screenshot is better as well. Note that you may have duplicate tests you will need to remove most likely.

Reply:
Updated it and removed the repeating test [here](https://github.com/cerner/terra-core/pull/1306/commits/b04c1661bc417ddbf31b7da23c28d95ec857e1df).